src/administration/troubleshooting.md

```diff
@@ -64,7 +64,25 @@ curl -H "Accept: application/activity+json" https://your-instance.com/comment/12
 
 Check that [federation is allowed on both instances](federation_getting_started.md).
 
-Also ensure that the time is accurately set on your server. Activities are signed with a timestamp, and will be discarded if it is off by more than 10 seconds.
+Also ensure that the time is correct on your server. Activities are signed with a timestamp, and will be discarded if it is off by more than one hour.
+
+It is possible that federation requests to `/inbox` are blocked by tools such as Cloudflare. The sending instance can find HTTP errors with the following steps:
+
+- Set `RUST_LOG=lemmy_federate=trace` for Lemmy
+- Reload the new configuration: `docker compose up -d`
+- Search for messages containing the target instance domain: `docker compose logs -f --tail=100 lemmy-federate |  grep lemm.ee -C 10`
+- If you use a [separate container for outgoing federation](horizontal_scaling.md), you need to apply the previous steps to that container only
+- You also may have to reset the fail count for the target instance (see below)
+
+### Reset federation fail count for instance
+
+If federation sending to a specific instance has been failing consistently, Lemmy will slow down sending using exponential backoff. For testing it can be useful to reset this and make Lemmy send activities immediately. To do this use the following steps:
+
+- Stop Lemmy, or specifically the container for outgoing federation `docker compose stop lemmy`
+- Enter SQL command line: `sudo docker compose exec postgres psql -U lemmy`
+- Find id of the target instance: `select * from instance where domain = 'lemm.ee';`
+- Reset failure count via SQL: `update federation_queue_state set fail_count = 0 where instance_id = 137189;`
```

Comment:
````suggestion
- Reset failure count via SQL:
```sql
update federation_queue_state
set fail_count = 0
from instance
where instance.id = federation_queue_state.instance_id
and instance.domain = 'lemm.ee'
```
````